tests/manage/z_cluster/nodes/test_az_failure.py

```diff
@@ -1,185 +0,0 @@
-import logging
-import random
-
-import pytest
-
-from ocs_ci.framework.pytest_customization.marks import aws_platform_required
-from ocs_ci.framework.testlib import ManageTest, tier4, tier4b
-from ocs_ci.ocs.exceptions import CommandFailed
-from ocs_ci.helpers import sanity_helpers
-
-logger = logging.getLogger(__name__)
-
-
-@tier4
-@tier4b
-@pytest.mark.polarion_id("OCS-1287")
-@aws_platform_required
-@pytest.mark.skip(reason="az blocking method need to be fixed")
-class TestAvailabilityZones(ManageTest):
-    """
-    test availability zone failure:
-    test stages:
-    1. Select availability zone
-    2. In this availability zone, backup instances original security groups
-    3. block availability zone by attaching security group with no permissions
-    4. validate - cluster functionality and health
-        2a. health check - warning or error
-        2b. create cephfs, create rbd, create pvc (validate_cluster)
-
-    5. restore availability zone access
-    6. validate - cluster functionality and health
-    """
-
-    @pytest.fixture(autouse=True)
-    def init_sanity(self):
-        """
-        init Sanity() object
-        """
-
-        self.sanity_helpers = sanity_helpers.Sanity()
-
-    @pytest.fixture()
-    def teardown(self, request, ec2_instances, aws_obj):
-        def finalizer():
-            current_sg = aws_obj.store_security_groups_for_instances(
-                self.instances_in_az
-            )
-            if self.original_sgs != current_sg:
-                aws_obj.restore_instances_access(
-                    self.security_group_id, self.original_sgs
-                )
-                logger.info(
-                    f"Access to EC2 instances {self.instances_in_az} has been restored"
-                )
-
-            if self.security_group_id in aws_obj.get_all_security_groups():
-                logger.info(f"Deleting: {self.security_group_id}")
-                aws_obj.delete_security_group(self.security_group_id)
-
-        request.addfinalizer(finalizer)
-
-    def test_availability_zone_failure(
```

Comment:
I would suggest keeping the test file and just replacing `test_availability_zone_failure` with `availability_zone_failure` which will cause this function not to be picked up by pytest collection